### DISCUSSION TITLE
How to get running on arm?

### DISCUSSION THREAD

**Discussion by ityonemo** [PlummersSoftwareLLC/Primes#717] [Q&A] [answered]
Hi!  In principle, the code I've written should run on arm, but I don't see it on the arm leaderboards.  Is there anything I should do to activate that?  Thanks!

**Comment by rbergen** [comment, marked as answer]
There's nothing specific that you should need to do to get your solution to run on ARM. It's more that you shouldn't do anything that makes it **not** run on that architecture. Specific things to consider are:

- The Docker base image needs to be available for the arm64 platform. Quite a few official images for various languages/runtimes do not support the platform (yet).
- The build steps that are specific to your solution should not require more hardware resources than what's available on the arm64 system(s) we use for benchmarking. At the moment, the one arm64 system we have at our disposal for benchmarking is an 8GB Raspberry Pi 4.

**Reply by ityonemo** [reply]
thanks for the specs on the build, I made an 8GB Arm QEMU image and found that the build process was OOMing.
